tslint.json

```diff
@@ -1,6 +1,6 @@
 {
   "rulesDirectory": [
-    "node_modules/tslint-microsoft-contrib",
```

Comment:
Note that running tslint will still error when `tslint-microsoft-contrib` is not installed. It is only specified as devDependency in package.json. Consider making it a peerDependency or even a full dependency.

Reply:
Agreed, `peerDependency` makes sense.